openssl/src/bn.rs

```diff
@@ -745,6 +745,25 @@ impl BigNumRef {
         }
     }
 
+    /// Places the result of `self^p mod m` in `self`. This makes it possible to avoid allocating
+    /// a new BigNum for each exponentiation. Useful for computations like `(((g^x_1 mod n)^x_2 mod n)^x_3 mod n)...`
```

Comment:
Can you match the style of the other method documentation? i.e. one sentence in the first paragraph, and a link to the OpenSSL documentation for the function we're calling.

Reply:
Yes.